RxCocoa/Runtime/_RXDelegateProxy.m

```diff
@@ -114,6 +114,14 @@ -(BOOL)voidDelegateMethodsContain:(SEL)selector {
     }
 }
 
+- (NSMethodSignature *)methodSignatureForSelector:(SEL)selector {
+    NSMethodSignature* signature = [super methodSignatureForSelector:selector];
```

Comment:
```suggestion
    NSMethodSignature *signature = [super methodSignatureForSelector:selector];
```

Reply:
I have added a test to verify the fix, and have removed the workaround for now. Does this seem good to you?
cc @SlashDevSlashGnoll @freak4pc@yan-zaitsev-hs @mkj-is

Reply:
Looks great, tested it and if I remove the fix, the test fails in the same way that the crash was.  👍   Thanks for adding that.

Reply:
Awesome!